sdk/scheduler/src/test/java/com/mesosphere/sdk/http/queries/EndpointsQueriesTest.java

```diff
@@ -180,7 +179,7 @@ private void testEndpoint(String expectedHostname) throws ConfigStoreException {
         assertEquals(200, response.getStatus());
         JSONObject json = new JSONObject((String) response.getEntity());
         assertEquals(json.toString(), 3, json.length());
-        assertEquals("vip1.svc-name.l4lb.thisdcos.directory:5432", json.get("vip"));
+        assertEquals("vip1.svc-name.vip.tld:5432", json.get("vip"));
```

Comment:
There seems to be a dependence between this test and some other setup since both the autoIP and VIP TLDs had to change.